src/host/_stream.cpp

```diff
@@ -1065,7 +1065,7 @@ using Microsoft::Console::VirtualTerminal::StateMachine;
 // - S_OK if successful.
 // - S_OK if we need to wait (check if ppWaiter is not nullptr).
 // - Or a suitable HRESULT code for math/string/memory failures.
-[[nodiscard]] HRESULT WriteConsoleWImplHelper(IConsoleOutputObject& context,

```

Comment:
Yeah I guess there was an idea to try to disguise all the BS inside SCREEN_INFORMATION and the InputBuffer at some point but it never panned out. So good removal.